html/canvas/element/text-styles/META.yml

```diff
@@ -22,3 +22,13 @@ links:
           subtest: System fonts must be computed to explicit values
         - test: 2d.text.font.parse.complex.html
           subtest: 'Canvas test: 2d.text.font.parse.complex'
+    - product: chrome
+      url: https://crbug.com/1066953
+      results:
+        - test: 2d.text.draw.baseline.hanging.html
+        - test: 2d.text.draw.baseline.ideographic.html
+        - test: 2d.text.draw.space.collapse.end.html
```

Comment:
These *.space.collapse.* tests do say "Space characters at the start of a line are collapsed (per CSS)" and while perhaps not identical to the problem in 2d.text.measure.width.space.html, let's keep them, it seems related enough.